Documentation/common-problems.md

```diff
@@ -4,22 +4,66 @@ weight: 78
 indent: true
 ---
 
-# Common Problems
+# Common Issues
 
 Many of these problem cases are hard to summarize down to a short phrase that adequately describes the problem. Each problem will start with a bulleted list of symptoms. Keep in mind that all symptoms may not apply depending upon the configuration of the Rook. If the majority of the symptoms are seen there is a fair chance you are experiencing that problem.
 
 If after trying the suggestions found on this page and the problem is not resolved, the Rook team is very happy to help you troubleshoot the issues in their Slack channel. Once you have [registered for the Rook Slack](https://rook-slackin.herokuapp.com/), proceed to the General channel to ask for assistance.
 
 ## Table of Contents
+- [Troubleshooting Techniques](#troubleshooting-techniques)
 - [Pod using Rook storage is not running](#pod-using-rook-storage-is-not-running)
 - [Cluster failing to service requests](#cluster-failing-to-service-requests)
 - [Only a single monitor pod starts](#only-a-single-monitor-pod-starts)
 - [OSD pods are failing to start](#osd-pods-are-failing-to-start)
 - [Node hangs after reboot](#node-hangs-after-reboot)
 
 # Troubleshooting Techniques
-One of the first things that should be done is to start the [rook-tools pod](./toolbox.md) as described in the Toolbox section. Once the pod is up and running one can `kubectl exec` into the pod to execute Ceph commands to evaluate that current state of the cluster. Here is a list of commands that can help one get an understanding of the current state.
+There are two main categories of information you will need to investigate issues in the cluster:
+1. Kubernetes status and logs
+1. Ceph status
+
+## Kubernetes Tools
+Kubernetes status will be the first line of investigating when something goes wrong with the cluster. Here are a few artifacts that would be helpful to gather:
+- Rook pod status: 
+  - `kubectl get pod -n rook -o wide`
+  - `kubectl get pod -n rook-system -o wide`
+- Logs for Rook pods
```

Comment:
we already have information about collecting logs, consider consolidating: https://github.com/rook/rook/blob/master/Documentation/advanced-configuration.md#log-collection

Reply:
Right, thanks for the reminder. What if I put a link to it here? I was trying to keep these instructions as simple as possible, and a script is too fancy for me.